flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -488,6 +488,23 @@ public enum SchedulerType {
                                             code(SchedulerExecutionMode.REACTIVE.name()))
                                     .build());
 
+    @Documentation.Section(Documentation.Sections.EXPERT_SCHEDULING)
+    public static final ConfigOption<Long> SCHEDULER_SCALING_INTERVAL_MIN =
+            key("jobmanager.adaptive-scheduler.scaling-interval.min")
+                    .longType()
+                    .defaultValue(30L) // favor lower scaling-interval.min for more reactive
+                    // rescaling and let the user increase the value for high workloads
+                    .withDescription(
+                            "Determines the minimum time (in seconds) between scaling operations in reactive mode.");
+
+    @Documentation.Section(Documentation.Sections.EXPERT_SCHEDULING)
+    public static final ConfigOption<Long> SCHEDULER_SCALING_INTERVAL_MAX =
+            key("jobmanager.adaptive-scheduler.scaling-interval.max")
+                    .longType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Allow the user to configure the time (in seconds) after which a scaling operation is triggered regardless if the requirements are met");
```

Comment:
These 2 option types are better to update from long to Duration.

Reply:
:+1:

Reply:
done